appinventor/appengine/build.xml

```diff
@@ -379,6 +379,7 @@
         <pathelement location="${lib.dir}/guava/guava-18.0.jar" />
         <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
         <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
+        <pathelement location="${lib.dir}/moxieapps/org.moxieapps.gwt.uploader-1.1.0.jar" />
```

Comment:
Is there an alternative library?



== Hal Abelson
hal@mit.edu
Prof. of Comp. Sci. and Eng.
MIT Dept. of Elec. Eng. and Comp. Sci.



On Tue, May 3, 2016 at 8:26 PM, Jeffrey Schiller <notifications@github.com>
wrote:

> In appinventor/appengine/build.xml
> <https://github.com/mit-cml/appinventor-sources/pull/763#discussion_r61978088>
> :
>
> > @@ -379,6 +379,7 @@
> >          <pathelement location="${lib.dir}/guava/guava-18.0.jar" />
> >          <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
> >          <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
> > +        <pathelement location="${lib.dir}/moxieapps/org.moxieapps.gwt.uploader-1.1.0.jar" />
>
> I'm concerned that a version of GWT in the future will be incompatible
> with this code. It is already 4 years old. I would rather not have this
> functionality, then have it and then have to remove it later.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-cml/appinventor-sources/pull/763/files/86c34d8bd5ec0b175a3b287035c4c6ceef7aeed8#r61978088>
>